src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,40 @@
+.catalog {
+  padding: 50px 40px;
+
+  display: grid;
+  grid-template-columns: repeat($column-count, $column-width);
+  justify-content: center;
+  gap: 46px 48px;
+
+  @media (max-width: 487px) {
+    & {
+      $column-count: 1;
+
+      grid-template-columns: repeat($column-count, $column-width);
+    }
+  }
+
+  @media (min-width: 488px) {
+    & {
+      $column-count: 2;
+
+      grid-template-columns: repeat($column-count, $column-width);
+    }
+  }
+
+  @media (min-width: 768px) {
+    & {
+      $column-count: 3;
```

Comment:
Defining the `$column-count` variable inside a media query makes it local to that scope. Consider defining it outside the media queries to ensure it's accessible globally and can be reused.